backend/api/__init__.py

```diff
@@ -78,10 +78,11 @@ def create_app(test_config=None):
     Migrate(app, db)
 
     # import and register blueprints
-    from api.views import main, message, box, fp, pm, docclass
+    from api.views import main, document, message, box, fp, pm, docclass
```

Comment:
I moved all Document views into their own file for clarity and consistency with other tables